content/spin/v2/quickstart.md

```diff
@@ -132,6 +132,14 @@ Installing template http-ts...
 
 {{ startTab "Python" }}
 
+Ensure that you have Python 3.10 or later installed on your system. You can check your Python version by running:
```

Comment:
This is in a section called "Install a Template".  It would fit more naturally in the "Install the Tools" section which is where we talk about things like having the right Rust bits or installing TinyGo.